src/routes/(app)/select-namespace/+page.svelte

```diff
@@ -73,8 +80,20 @@
 </div>
 
 <style lang="postcss">
+  .search {
+    @apply mb-5 flex rounded-full border-[3px] border-gray-900 bg-white p-1 pr-4;
```

Comment:
I don't love the hard-coded values here. Should we just add to the values in Tailwind? That said, it's not a blocker at this moment.

Reply:
Extended our `tailwind.config` to include 
```js
      borderWidth: {
        '3': '3px',
      },
```